src/Components/GrowthChart/GrowthChart.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { GrowthChartBuilder } from "./GrowthChartBuilder";
+import { chartData } from "../../DataSets/ChartData";
+
+export const GrowthChart = () => {
+    const selectedChart = chartData["Weight-for-age GIRLS"]
```

Comment:
This can be removed if you choose to move the logic from the GrowthChartBuilder component to this one